typings/space.d.ts

```diff
@@ -659,18 +659,20 @@ export interface ContentfulSpaceAPI {
    * .then((space) => space.createTeamSpaceMembership('team_id', {
    *   admin: false,
    *   roles: [
-   *     {
+   *    {
+          sys: {
    *       type: 'Link',
    *       linkType: 'Role',
    *       id: '<role_id>'
-   *     }
+   *      }
+   *    }
    *   ],
    * }))
    * .then((teamSpaceMembership) => console.log(teamSpaceMembership))
    * .catch(console.error)
    * ```
    */
-  createTeamSpaceMembership(data: TeamSpaceMembershipProps): Promise<TeamSpaceMembership>
+  createTeamSpaceMembership(teamId: string, data: TeamSpaceMembershipProps): Promise<TeamSpaceMembership>
```

Comment:
@anho yep, it was fixed in `master` branch. It's shown as diff here because I've merged master to this branch to streamline changes to `experimental` branch.